index.js

```diff
@@ -349,7 +349,10 @@ module.exports = async function(options) {
       }
     }
     function test(name) {
-      if (!validSteps.includes(name)) {
+      // Projects that have different theme modules activated at different times
+      // are a frequent source of false positives for this warning, so ignore
+      // seemingly unused modules with "theme" in the name
+      if (!validSteps.includes(name) || (!name.match(/theme/))) {
```

Comment:
I think this should be more explicit. Like an option to mark that it's a theme. Or that it will not be instantiated. We need something we can document, and saying to include the word "theme" in the module name doesn't seem an industry best practice.
TLDR; I think this should be an option instead.